File: openweathermap-connector-java/src/com/openweathermap/kafka/connect/model/Wind.java
package com.openweathermap.kafka.connect.model;

import com.google.gson.annotations.SerializedName;

public class Wind {

    @SerializedName("speed")
    private Float speed;

    @SerializedName("deg")
    private Integer deg;

    public Wind() {}

    public Wind(Float speed, Integer deg) {
        this.speed = speed;
        this.deg = deg;
    }

    public Float getSpeed() {
        return speed;
    }

    public void setSpeed(Float speed) {
        this.speed = speed;
    }

    public Integer getDeg() {
        return deg;
    }

    public void setDeg(Integer deg) {
        this.deg = deg;
    }
}

File: openweathermap-connector-scala/src/com/openweathermap/kafka/connect/model/Wind.scala
package com.openweathermap.kafka.connect.model

import com.google.gson.annotations.SerializedName
import scala.beans.{BeanProperty, BooleanBeanProperty}
import scala.collection.JavaConversions._

class Wind {

  @SerializedName("speed")
  @BeanProperty
  var speed: java.lang.Float = _

  @SerializedName("deg")
  @BeanProperty
  var deg: java.lang.Integer = _

  def this(speed: java.lang.Float, deg: java.lang.Integer) = {
    this()
    this.speed = speed
    this.deg = deg
  }

}
